VideoWeb/VideoWeb/ClientApp/src/app/services/video-filter.service.ts

```diff
@@ -162,9 +162,13 @@ export class VideoFilterService {
         }
     }
 
+    isFeatureEnabled() {
+        return this._enableVideoFilters;
+    }
+
     doesSupportVideoFiltering() {
         const allowedBrowser = !this.deviceTypeService.getBrowserName().includes(browsers.Safari);
-        return this.enableVideoFilters && allowedBrowser && !this.deviceTypeService.isTablet();
+        return this._enableVideoFilters && allowedBrowser && !this.deviceTypeService.isTablet();
```

Comment:
I think this currently would allow mobile users to use custom backgrounds. Should it be.
```suggestion
        return this._enableVideoFilters && allowedBrowser && this.deviceTypeService.isDesktop();
```

Reply:
Thanks. That's correct. There's overlapping features which have probably made it unclear which devices are being supported.